### PR TITLE
Attribute examples

### PR DESCRIPTION
**Pull request by Ushagb**
Examples for Attribute directives - ngClass and ngStyle

### CODE REVIEW

File: src/app/attribute/attribute-directive.component.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.primary{
+    color:red;
+}
+.big{
+    font-size: 30px;
```
Comment (satgrocks) [inline src/app/attribute/attribute-directive.component.css]: change the font size to 35

File: src/app/attribute/attribute-directive.component.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!-- ngClass  -->
+<div [ngClass]="['primary', 'big']">Sample Text</div>
+
+<!-- We can give condition in { } parenthesis -->
+
+<input type="checkbox" [(ngModel)]="applyColor">
+<div [ngClass]="{'primary':applyColor}">Hello World</div>
+
+<!-- ngStyle -->
+<p [ngStyle]="{'color':'green', 'font-size':'40px' }">Hello This is Usha</p>
```
Comment (satgrocks) [inline src/app/attribute/attribute-directive.component.html:10]: use "\<div>" instead of "\<p>"

**Review by satgrocks — APPROVED** [review]

**Review by satgrocks — CHANGES_REQUESTED** [review]
make changes as suggested

File: src/app/attribute/attribute-directive.component.css (hunk 1 above)
Comment (Ushagb) [reply src/app/attribute/attribute-directive.component.css]: Okey

**Pull request merged**